tests/pytests/functional/states/win_lgpo/test_audit_settings_state.py

```diff
@@ -0,0 +1,124 @@
+    # Short Name: SceNoApplyLegacyAuditPolicy
+    from tests.support.sminion import create_sminion
+
+    salt_minion = create_sminion()
+    test_setting = "Disabled"
+    pre_security_setting = salt_minion.functions.lgpo.get_policy(
+        policy_name="SceNoApplyLegacyAuditPolicy", policy_class="machine"
+    )
+    pre_audit_setting = salt_minion.functions.lgpo.get_policy(
+        policy_name="Audit Account Management", policy_class="machine"
+    )
+    try:
+        if pre_security_setting != test_setting:
+            salt_minion.functions.lgpo.set_computer_policy(
+                name="SceNoApplyLegacyAuditPolicy", setting=test_setting
+            )
+            assert (
+                salt_minion.functions.lgpo.get_policy(
+                    policy_name="SceNoApplyLegacyAuditPolicy", policy_class="machine"
+                )
+                == test_setting
+            )
+        yield
+    finally:
+        salt_minion.functions.lgpo.set_computer_policy(
+            name="SceNoApplyLegacyAuditPolicy", setting=pre_security_setting
+        )
+        salt_minion.functions.lgpo.set_computer_policy(
+            name="Audit Account Management", setting=pre_audit_setting
+        )
+
+
+@pytest.fixture(scope="function")
+def clear_policy():
+    # Ensure the policy is not set
+    test_setting = "No auditing"
+    win_lgpo_module.set_computer_policy(
+        name="Audit account management", setting=test_setting
+    )
+    assert (
+        win_lgpo_module.get_policy(
+            policy_name="Audit account management", policy_class="machine"
+        )
+        == test_setting
+    )
+
+
+@pytest.fixture(scope="function")
+def set_policy():
+    # Ensure the policy is set
+    test_setting = "Success"
+    win_lgpo_module.set_computer_policy(
+        name="Audit account management", setting=test_setting
+    )
+    assert (
+        win_lgpo_module.get_policy(
+            policy_name="Audit account management", policy_class="machine"
+        )
+        == test_setting
+    )
+
+
+def _test_auditing(setting):
+    """
+    Helper function to set an audit setting and assert that it was successful
+    """
+    computer_policy = {"Audit Account Management": setting}
+    win_lgpo_state.set_(name="junk", computer_policy=computer_policy)
+    # Clear the context so we're getting the actual settings from the machine
+    win_lgpo_module._get_secedit_data(refresh=True)
+    result = win_lgpo_module.get_policy(
+        policy_name="Audit Account Management", policy_class="machine"
+    )
+    assert result == setting
+
+
+def test_no_auditing(enable_legacy_auditing, set_policy):
+    _test_auditing("No auditing")
+
+
+def test_success(enable_legacy_auditing, clear_policy):
+    _test_auditing("Success")
+
+
+def test_failure(enable_legacy_auditing, clear_policy):
+    _test_auditing("Failure")
+
+
+def test_success_and_failure(enable_legacy_auditing, clear_policy):
+    _test_auditing("Success, Failure")
```

Comment:
same here, can we test for `"Success and Failure"`

Reply:
Same here